test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -23,7 +23,8 @@ def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
         pytest.skip("Currently skipping for TF2.3.0 on p2.8xlarge until the issue is fixed")
     if is_tf1(training):
         pytest.skip("Currently skipping for TF1 until the issue is fixed")
-    run_smdebug_test(training, ec2_connection, region, docker_executable="nvidia-docker", container_name="smdebug-gpu")
+    run_smdebug_test(training, ec2_connection, region, docker_executable="nvidia-docker", container_name="smdebug-gpu",
+                     is_large_shm="p2.8xlarge" in SMDEBUG_EC2_GPU_INSTANCE_TYPE)
```

Comment:
Please get the `ec2_instance_type` as an input to the `test_smdebug_gpu` function, and check if `ec2_instance_type in ["p2.8xlarge"]`, so that the instance type for the current test is checked, and not all instance types for all parametrized tests.

Reply:
@saimidu let's handle the bigger change in the PR below (I am doing essentially this).

https://github.com/aws/deep-learning-containers/pull/623/files